util/src/test/java/io/kubernetes/client/util/credentials/TokenFileAuthenticationTest.java

```diff
@@ -63,19 +65,6 @@ void tokenProvided() throws ApiException {
         1,
         getRequestedFor(urlPathEqualTo("/api/v1/pods"))
             .withHeader("Authorization", equalTo("Bearer token1")));
-
-    this.auth.setFile(SERVICEACCOUNT_TOKEN2_PATH);
```

Comment:
these unit tests are no longer needed as it was originally for testing token expiry/rotation behavior. now it's decoupled/delegated to the RefreshAuthentication class.